src/App.tsx

```diff
@@ -1,156 +1,70 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useCallback, useEffect, useRef, useState,
+} from 'react';
 
-const USER_ID = 0;
+import { Title } from './components/Title/Title';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
+import { Content } from './components/Content/Content';
+import { Todo } from './types/Todo';
+import { ErrorMessages } from './enums/ErrorMessages';
+import { getTodos } from './api/todos';
+
+const USER_ID = 11076;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [showError, setShowError] = useState(false);
+
+  const errorMessage = useRef<ErrorMessages>(ErrorMessages.noError);
```

Comment:
We can combine it into a single errorMessage state using useState, storing the error message, and checking if ErrorMessages.noError means that the error doesn't exist.

Reply:
When we add `errorMessage` to state, we'll have changing error message in component `ErrorNotification.tsx` immediately, so we will have problem with animation. 

https://github.com/mate-academy/react_todo-app-loading-todos/assets/83166880/98adcf43-ce37-4b03-84c6-a8100864521b

Reply:
Or I can just add state with` {message: string, showMessage: boolean }` but it add new rerender